examples/flutter_gallery/lib/demo/material/full_screen_dialog_demo.dart

```diff
@@ -106,6 +106,7 @@ class FullScreenDialogDemoState extends State<FullScreenDialogDemo> {
   DateTime _toDateTime = new DateTime.now();
   bool _allDayValue = false;
   bool _saveNeeded = false;
+  String _eventName = 'New event';
```

Comment:
This ends up being a little confusing because the app's title doesn't match the (initially empty) event name text field.

It might be better if the app's title was initially ``"<Event Name TBD>"``, or something better if you can think of it :-). You could use _eventName == null to indicate that a name hadn't been specified yet.